sdk/log/simple.go

```diff
@@ -55,7 +98,20 @@ func (s *SimpleProcessor) OnEmit(ctx context.Context, r *Record) error {
 		simpleProcRecordsPool.Put(records)
 	}()
 
-	return s.exporter.Export(ctx, *records)
+	err := s.exporter.Export(ctx, *records)
+
+	if s.selfObservabilityEnabled {
+		attrs := []attribute.KeyValue{
```

Comment:
We can predefine and allocate memory in advance, which can save the overhead of secondary memory allocation.